packages/notebook-extension/src/index.ts

```diff
@@ -2485,6 +2503,7 @@ function addCommands(
   });
   commands.addCommand(CommandIDs.moveUp, {
     label: trans.__('Move Cells Up'),
+    caption: trans.__('Move cell up'),
```

Comment:
```suggestion
    caption: trans.__('Move these cells up'),
```

Reply:
Just to follow up here, I think that ideally it would be "Move X cells up" where X is the number of selected cells.

Reply:
> Just to follow up here, I think that ideally it would be "Move X cells up" where X is the number of selected cells.

@krassowski I thought about precising the number of cells. This can be done if you all agree @fcollonval @JasonWeill

Reply:
Sounds good:

* Move cell up (0–1 cell selected)
* Move X cells up (2+ cells selected)